copy.go

```diff
@@ -250,7 +250,7 @@ func copyGraph(ctx context.Context, src content.ReadOnlyStorage, dst content.Sto
 		if err := region.Start(); err != nil {
 			return err
 		}
-		return copyNode(ctx, proxy.Cache, dst, desc, opts)
+		return copyNode(ctx, proxy, dst, desc, opts)
```

Comment:
If the content is not in the proxy cache, shouldn't copy from the source directly just like the copy operations for leaf node?

Reply:
Copying from the source is the safiest choice, but in that case `proxy` won't be used at all, is what we expected?

Reply:
A manifest needs not to be cached if its successors are externally provided.